static/js/components/CompactCommentList.jsx

```diff
@@ -71,6 +71,7 @@ const CompactCommentList = ({
           lastName={author.last_name}
           username={author.username}
           gravatarUrl={author.gravatar_url}
+          avatar={author?.avatar}
```

Comment:
No sure if the question mark makes sense here. If there is no `author` it would crash on the lines right above it.

Reply:
good point.